examples/simultaneous_translation/docs/evaluation.md

```diff
@@ -1,10 +1,10 @@
 # Introduction to the evaluation interface
-The simultaneous translation models from the shared task participants are evaluated under a server-client protocol. The participants are required to plug in their own model API in the protocol, and submit a Docker image.
+The simultaneous translation models from the shared task participants are evaluated under a server-client protocol. 
+The participants are required to plug in their own model API in the protocol, and submit a Docker image.
```

Comment:
Docker image or Docker file? I think image is what it built but they don't submit the image, just a way for us to build it and run it.